src/bluesearch/database/article.py

```diff
@@ -125,13 +125,13 @@ def uid(self) -> Optional[str]:
         return generate_uid((self.pubmed_id, self.pmc_id, self.doi))
 
 
-class PMCXMLParser(ArticleParser):
-    """Parser for PubMed Central XML files using the JATS Journal Publishing DTD.
+class JATSXMLParser(ArticleParser):
+    """Parser for JATS XML files.
```

Comment:
Maybe we can list here or in the CLI help (or in both?) what the potential article sources are that support this schema. I guess in this case it would be PMC, bioRxiv, medRxiv. Just to make it easier for whoever is reading the code / the CLI help to understand what kind of files were intended to be used as an input.